main.go

```diff
@@ -18,6 +20,101 @@ import (
 	tar "kubevirt.io/containerdisks/pkg/build"
 )
 
+const (
+	diskPath             string = "./tmp/disk.img.gz"
+	diskPathDecompressed string = "./tmp/disk.img"
+	diskPathConverted    string = "./tmp/disk.qcow2"
+)
+
+func apply(vmName string) error {
+	log.Println("Applying VirtualMachineExport object...")
+
+	yaml := fmt.Sprintf(`
+apiVersion: export.kubevirt.io/v1alpha1
+kind: VirtualMachineExport
+metadata:
+  name: %s
+spec:
+  source:
+    apiGroup: "kubevirt.io"
+    kind: VirtualMachine
+    name: %s
+`, vmName, vmName)
+	cmd := exec.Command("kubectl", "apply", "-f", "-")
+	cmd.Stdin = strings.NewReader(yaml)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Run(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func download(vmName, volumeName string) error {
```

Comment:
I would rename this function to something like downloadVMDisk

Reply:
Done.